proto/public/api.proto

```diff
@@ -66,9 +66,7 @@ message MetricSeries {
 }
 
 message MetricMetadata {
-  string targetPod = 1;
```

Comment:
Can condense the existing fields to make use of the newly available field numbers.

Reply:
I didn't do this because the gRPC docs said it was considered harmful. But if we're not concerned about compatibility, can change.